src/views/gallery.coffee

```diff
@@ -62,6 +62,7 @@ class GalleryView
     @viewport = $( (options || {}).viewport || @el)
     @forceFullWidth = Boolean(options.forceFullWidth || false)
     $(window).on 'resize', _.debounce(@layout, 100)
+    window.matchMedia('print').addListener(@layout)
```

Comment:
Do you think it's worth to play safe here and use [the existential operator](https://coffeescript.org/#existential-operator)?

```suggestion
    window.matchMedia?('print').addListener(@layout)
```

Reply:
Makes sense :+1: